components/ui/NavBar.tsx

```diff
@@ -13,21 +44,60 @@ export const Navbar = ({ items }: { items: NavItem[] }) => {
         >
           <h1 className="text-xl md:text-3xl">{siteConfig.name}</h1>
         </Link>
+        <nav className="flex text-xl md:text-2xl">
         {items?.length ? (
-          <ul className="flex text-xl md:text-2xl">
-            {items?.map((item, idx) => (
-              <li key={idx} className="px-4">
-                <Link
-                  href={item.href}
-                  className="text-white text-opacity-75 transition duration-150 ease-in-out hover:text-opacity-100"
-                >
-                  {item.title}
-                </Link>
-              </li>
-            ))}
-          </ul>
+            items?.map((item, idx) => (
+              <Link
+                key={idx}
+                href={item.href}
+                className="text-white text-opacity-75 transition duration-150 ease-in-out hover:text-opacity-100 mr-2"
+              >
+                {item.title}
+              </Link>
+            ))
         ) : null}
+        {}
+        {!loggedIn && <button onClick={() => setOpenSignInModal(true)} className="text-grey-300 w-10 h-10 ml-4 cursor-pointer">
+          <svg width="30" height="31" viewBox="0 0 30 31" fill="none" xmlns="http://www.w3.org/2000/svg">
+            <g clip-path="url(#clip0_393_10135)">
```

Comment:
It's a bit messy to have these clip-paths in inline SVGs. I need to clean up the vectors in Figma so we can export clean ones. Also, can we refer to SVG asset files and have them automatically pulled in during compile/build?

Reply:
Done in ed38633f9b207f81ef94253542245d3e6c91529a